src/classes/GE_DefaultTemplate.cls

```diff
@@ -137,13 +137,13 @@ public with sharing class GE_DefaultTemplate {
         populateDevNameMapping(updatedFieldDevNameMap, updatedObjectDevNameMap);
```

Comment:
We have an unused variable at the top of this class `ELEMENT_TYPE_FIELD`, can we remove it ?

Also the `buildDefaultTemplate` method could use some refactoring because of its size, but I'm not sure if we want to get into it for this WI. What do you think ?

Reply:
👍  for removing `ELEMENT_TYPE_FIELD`.

What do ya'll think about adding the refactor of specific methods to our refactor backlog quip doc?

Reply:
Agree with removing the unused variable, and with the need to refactor the buildDefaultTemplate method.  

I'll push the update to remove the variable now and I like the idea of adding specific method to our refactor backlog document.  This WI is really specifically geared toward organization/re-location so I want to shy away from refactoring the internals of any methods here.  And I think having these methods queued up in the doc will help us to identify them as good candidates for refactoring when doing design docs for features and changes that will touch these same classes again in the future, which is when I think we should propose/implement the internal refactoring of this and other similar methods.

Reply:
I enthusiastically support the above decisions - whenever a potential refactor CAN be isolated to a particular method, that would be a great way to break up the work into small-ish sections and I think clarify the possible regression impacts as well. Nothing new to contribute beyond that. Y'all are smart!

Reply:
👍🏽✅